packages/shared/components/AssetTile.svelte

```diff
@@ -1,12 +1,14 @@
 <script lang="typescript">
+    import { IAsset } from '@core/wallet'
```

Comment:
You should be able to import both of these in one `import` statement as long as the barrel files are correct.

Reply:
Done